cli/args/package_json.rs

```diff
@@ -227,6 +227,26 @@ mod test {
     );
   }
```

Comment:
Oh, it looks like this was previously purposefully done, but we should try to be as npm compatible as possible.